packages/augur-tools/src/flash/scripts.ts

```diff
@@ -444,14 +468,82 @@ export function addScripts(flash: FlashSession) {
     },
   });
 
+  flash.addScript({
+    name: 'contribute-to-tentative-winning-outcome',
+    options: [
+      {
+        name: 'marketId',
+        abbr: 'm',
+        description:
+          'market to contribute REP to its tentative winning outcome',
+        required: true,
+      },
+      {
+        name: 'outcome',
+        abbr: 'o',
+        description:
+          'outcome of the market, non scalar markets 0,1,3,... for scalar markets use price',
+        required: true,
+      },
+      {
+        name: 'amount',
+        abbr: 'a',
+        description: 'amount of REP to dispute with, use display value',
+        required: false,
```

Comment:
I think this has to be `true` due to line #510, where you do `new BigNumber(amount)`. A missing `amount` is going to be an empty string (I think?) not `'0'`.

Reply:
ok, prob copy paste error, will check it out

Reply:
good catch, pushed change